### PR TITLE
Important Note to a resource where are all the options #5072

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added important link note.
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

JGO

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: devices/surface/surface-dock-firmware-update.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ You can use Windows Installer commands (Msiexec.exe) to deploy Surface Dock Firm
 
 For more information, refer to [Command line options](https://docs.microsoft.com/windows/win32/msi/command-line-options) documentation.
 
+> [!IMPORTANT]
+> Also, if you want to keep your surface dock updated using any other method, please [follow the link](https://support.microsoft.com/help/4023478/surface-update-your-surface-dock).
```
Comment (JohanFreelancer9) [inline devices/surface/surface-dock-firmware-update.md]:
```suggestion
> If you want to keep your Surface Dock updated using any other method, please see [Update your Surface Dock](https://support.microsoft.com/help/4023478/surface-update-your-surface-dock) for details.
```

**Comment by j0rt3g4** [comment]
@JohanFreelancer9  done :)

**Comment by illfated** [comment]
Third attempt, ref. issue ticket #5072 .

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. There is also an additional question from the customer related to this PR based on https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5072#issuecomment-537819148 which we hope you can help provide your insights on. 

Thank you.

cc: @kenwith

**Comment by illfated** [comment]
Please note that there has been at least 7 updates to the affected file (https://github.com/MicrosoftDocs/windows-itpro-docs/commits/public/devices/surface/surface-dock-firmware-update.md) since October 3, 2019, so you may want to rebase this file or merge the public branch into this PR branch to solve the conflicts. Note also that this page may need additional content modification as discussed in issue ticket [**#4984** (**Microsoft Surface Dock Updater Doesn't Exist**)](https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4984) .

**Comment by illfated** [comment]
See also https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5098 by @VLG17 affecting this file.

**Comment by j0rt3g4** [comment]
#substituted by PR #5328 

(I had to fork winIT docs again, so I didn't have access anymore to the PR #5098  anymore. Created a new one.

**Pull request closed without merging**